### PR TITLE
Updates audit cookbook instructions

### PR DESCRIPTION
**Pull request by kagarmoe**

### CODE REVIEW

File: chef_master/source/perform_compliance_scan.rst [hunk 1]
```diff
@@ -11,102 +11,40 @@ Perform a Compliance Scan in Chef Automate
...
+The examples shown in this topic are meant to provide a quick way for you to see compliance data show up in Chef Automate. You could also wrap the audit cookbook in an existing cookbook, but this example will simply use the default recipe in the audit cookbook to run a profile against a node in your cluster. For more information and examples on how to use the audit cookbook, see the `audit project repo in GitHub <https://github.com/chef-cookbooks/audit>`_.
 
 If your workflow requires the use of the standalone Chef Compliance server, or you are using a previous version of Chef Automate (older than 0.8.5), see :doc:`/integrate_compliance_server_chef_automate` for information on how to use the audit cookbook to scan your nodes.
 
 .. note:: Remote scanning capabilities currently part of Chef Compliance and will be available in Chef Automate in the future.
 
-.. tag beta_note
```
Comment (kagarmoe) [inline chef_master/source/perform_compliance_scan.rst]: The beta note needs moved over -->

File: chef_master/source/perform_compliance_scan.rst [hunk 2]
```diff
@@ -180,7 +163,7 @@ A truncated version of the bootstrap output is shown below:
    localhost [2017-05-17T21:11:07+00:00] INFO: Running report handlers
    localhost [2017-05-17T21:11:07+00:00] WARN: Format is json
    localhost [2017-05-17T21:11:07+00:00] INFO: Initialize InSpec 1.24.0
-   localhost [2017-05-17T21:11:08+00:00] INFO: Running tests from: [{:name=>"DevSec Linux Security Baseline", :compliance=>"Dave/linux-baseline"}, {:name=>"DevSec SSH Baseline", :compliance=>"Dave/ssh-baseline"}]
+   localhost [2017-05-17T21:11:08+00:00] INFO: Running tests from: [{:name=>"DevSec Linux Security Baseline", :compliance=>"User/linux-baseline"}, {:name=>"DevSec SSH Baseline", :compliance=>"User/ssh-baseline"}]
```
Comment (kagarmoe) [inline chef_master/source/perform_compliance_scan.rst]: Change names of profiles

File: chef_master/source/perform_compliance_scan.rst [hunk 3]
```diff
@@ -11,102 +11,40 @@ Perform a Compliance Scan in Chef Automate
 
 .. end_tag
 
+.. tag compliance_beta
+
+.. important:: The new compliance functionality described in this topic is currently in Beta. To enable compliance reporting in the Chef Automate UI, navigate to the **Nodes** tab, make sure your cursor is not in any text box or field, and type ``beta``. A new ``Compliance`` tab should appear in the top-level menu of the UI. Note: Enabling the ``Compliance`` tab will allow you to view only new compliance scan data, not historical data.
```
Comment (davidwrede) [inline chef_master/source/perform_compliance_scan.rst]: As of Chef Automate 1.5.46, the integrated compliance functionality is GA, so we shouldn't need to add this note back in.

File: chef_master/source/perform_compliance_scan.rst [hunk 4]
```diff
@@ -11,102 +11,40 @@ Perform a Compliance Scan in Chef Automate
...
-
-.. end_tag
-
 Prerequisites
 -----------------------------------------------------
 
 The following are required when using the built-in compliance capabilities of Chef Automate:
 
 * Chef client 12.16.42 or later must be installed on your nodes
-* Chef Automate server 1.5.46 or later (the compliance view was previously enabled through a ``beta`` flag in 0.8.5)
```
Comment (jerryaldrichiii) [inline chef_master/source/perform_compliance_scan.rst]: I think we should use this line but remove the bits in parenthesizes.

File: chef_master/source/perform_compliance_scan.rst [hunk 5]
```diff
@@ -11,102 +11,40 @@ Perform a Compliance Scan in Chef Automate
...
-        Uploading audit        [4.0.0]
-        Uploading compat_resource [12.19.0]
-        Uploading simple_cookbook [0.1.0]
-        Uploaded all cookbooks.
-
 Configure Data Collection on Chef server
 -------------------------------------------------------
 
-To proxy node data through Chef server to Chef Automate, you must update the ``/etc/opscode/chef-server.rb`` file on your Chef server. This is needed for convergence status and general node data, but it is also true for proxying audit run data from nodes back to Chef Automate.
+To send node data through Chef server to Chef Automate, you must update the ``/etc/opscode/chef-server.rb`` file on your Chef server. This is needed for convergence status and general node data, but it is also true for sending audit run data from nodes back to Chef Automate.
```
Comment (jerryaldrichiii) [inline chef_master/source/perform_compliance_scan.rst]: `convergence` or `converge` status? I prefer the latter.

**Comment by jerryaldrichiii** [comment]
There needs to be a section for configuring Data Collection on the Automate Server. See:
http://blog.jerryaldrichiii.com/chef_automate/2017/08/04/compliance-using-the-audit-cookbook.html#configure-data-collection

File: chef_master/source/perform_compliance_scan.rst [hunk 6]
```diff
@@ -131,46 +69,91 @@ Before you can see if your nodes are compliant, you need to have the profiles yo
...
 
    .. note:: Your profiles must be archived as tarballs with a ``.tar.gz`` extension.
 
-Use the audit cookbook on nodes in your cluster
--------------------------------------------------------
+Using the Audit Cookbook
+------------------------------------------------------
 
-Once you have the audit cookbook installed on your Chef server and your profiles are ready to use in Chef Automate, you must add the ``audit::default`` recipe to your nodes' run-lists. The audit cookbook can be reused for all nodes because all node-specific configuration is done through Chef attributes.
+The recommended method for using the audit cookbook is via a wrapper cookbook. A wrapper cookbook is a regular cookbook that includes recipes from other cookbooks. In case the wrapper cookbook will use ``include audit::default``.
```
Comment (jerryaldrichiii) [inline chef_master/source/perform_compliance_scan.rst]: `In this case`? Also I would change `include audit::default` to just `audit::default` or `the default recipe of the audit cookbook`

File: chef_master/source/perform_compliance_scan.rst [hunk 7]
```diff
@@ -131,46 +69,91 @@ Before you can see if your nodes are compliant, you need to have the profiles yo
...
-           "compliance": "<username>/ssh-baseline"
-         }
-       ],
-       "reporter": "chef-server-automate"
-     }
-   }
+  chef generate cookbook mycompany_audit
 
-.. note:: The ``audit`` cookbook also requires a time synchronization policy to be in place, such as Network Time Protocol (NTP).
+Place the following in mycompany_audit/metadata.rb:
```
Comment (jerryaldrichiii) [inline chef_master/source/perform_compliance_scan.rst]: Maybe put file paths in `` blocks?

File: chef_master/source/perform_compliance_scan.rst [hunk 8]
```diff
@@ -131,46 +69,91 @@ Before you can see if your nodes are compliant, you need to have the profiles yo
...
+
+.. code-block:: ruby
+
+  default['audit']['reporter'] = 'chef-server-automate'
+  case node['os']
+  when 'linux'
+    default['audit']['profiles'] = [
+     {
+        'name': 'DevSec Linux Security Baseline',
+        'compliance': 'admin/linux-baseline'
```
Comment (jerryaldrichiii) [inline chef_master/source/perform_compliance_scan.rst:129]: In order to use `admin/` they will need to make sure that these profiles are downloaded as the `admin` user. See Step 1:
http://blog.jerryaldrichiii.com/chef_automate/2017/08/04/compliance-using-the-audit-cookbook.html#download-the-necessary-profiles

File: chef_master/source/perform_compliance_scan.rst [hunk 9]
```diff
@@ -131,46 +69,91 @@ Before you can see if your nodes are compliant, you need to have the profiles yo
...
+  berks install && berks upload
+
+Collect Compliance Scan Data
++++++++++++++++++++++++++++++++++++++++
+You can add the ``mycompany_wrapper::default`` recipe to an existing run-list; however, in the example below, we will bootstrap a node with the ``mycompany_wrapper::default`` recipe and run a series of baseline checks against a new node.
 
 .. code-block:: bash
 
-   $ knife bootstrap localhost --ssh-port 2201 --ssh-user vagrant --sudo --identity-file /Users/<username>/Repos/chef-test/.vagrant/machines/node1-ubuntu/virtualbox/private_key --node-name node1-ubuntu --json-attribute-file files/test_audit.json --run-list 'recipe[audit::default]'
+   knife bootstrap localhost --ssh-port 2201 --ssh-user vagrant --sudo --identity-file /Users/<username>/Repos/chef-test/.vagrant/machines/node1-ubuntu/virtualbox/private_key --node-name node1-ubuntu --run-list 'recipe[mycompany_wrapper::default]'
```
Comment (jerryaldrichiii) [inline chef_master/source/perform_compliance_scan.rst]: No need for such a complicated command. It should be:

```
knife bootstrap NODE_IP_OR_FQDN --ssh-user ADMIN_USERNAME --sudo --identity-file ~/.ssh/private_key.pem --run-list 'recipe[mycompany_wrapper::default]'
```

or:

```
knife bootstrap NODE_IP_OR_FQDN -x ADMIN_USERNAME -P ADMIN_PASSWORD --sudo -r 'recipe[mycompany_wrapper::default]'
```

Also, there are quite a few ways to bootstrap nodes. We should link: https://docs.chef.io/knife_bootstrap.html

File: chef_master/source/perform_compliance_scan.rst (hunk 2 above)
Comment (jerryaldrichiii) [inline chef_master/source/perform_compliance_scan.rst]: Should change the name to `admin/` to match our instructions above.

File: chef_master/source/perform_compliance_scan.rst [hunk 10]
```diff
@@ -189,9 +172,10 @@ A truncated version of the bootstrap output is shown below:
    localhost [2017-05-17T21:11:11+00:00] INFO: Report handlers complete
    localhost Chef Client finished, 0/2 resources updated in 18 seconds
 
+.. note:: The ``audit`` cookbook also requires a time synchronization policy to be in place, such as Network Time Protocol (NTP).
+
 View the results in Chef Automate
 -------------------------------------------------------
-
 When you go back to your Chef Automate UI under the **Compliance** tab, the **Reporting** dashboard should be visible. By alternating between **Node Status** and **Profile Status** views, you can view the scan results depending on which view is most important to you. The following shows some of the scan results on the bootstrapped node, "node1-ubuntu".
```
Comment (jerryaldrichiii) [inline chef_master/source/perform_compliance_scan.rst]: We shouldn't be instructing users to bootstrap vagrant instances or setting the node name. I would remove the `, "node1-ubuntu"` bit.

**Review by jerryaldrichiii — CHANGES_REQUESTED** [review]

**Comment by jerryaldrichiii** [comment]
Not part of your changes but I would like to see `Upload Profiles to Chef Automate` renamed to `Adding Profiles to Chef Automate` since we talk about uploading and downloading.

File: chef_master/source/perform_compliance_scan.rst (hunk 4 above)
Comment (kagarmoe) [reply chef_master/source/perform_compliance_scan.rst]: OK.

File: chef_master/source/perform_compliance_scan.rst (hunk 8 above)
Comment (kagarmoe) [reply chef_master/source/perform_compliance_scan.rst:129]: Thanks for pointing that out-

File: chef_master/source/perform_compliance_scan.rst (hunk 10 above)
Comment (kagarmoe) [reply chef_master/source/perform_compliance_scan.rst]: ok

File: chef_master/source/perform_compliance_scan.rst (hunk 5 above)
Comment (kagarmoe) [reply chef_master/source/perform_compliance_scan.rst]: OK.

File: chef_master/source/perform_compliance_scan.rst (hunk 7 above)
Comment (kagarmoe) [reply chef_master/source/perform_compliance_scan.rst]: ok

File: chef_master/source/perform_compliance_scan.rst (hunk 9 above)
Comment (kagarmoe) [reply chef_master/source/perform_compliance_scan.rst]: ok.

File: chef_master/source/perform_compliance_scan.rst (hunk 8 above)
Comment (kagarmoe) [reply chef_master/source/perform_compliance_scan.rst:129]: Done.

**Pull request merged**